<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientSimuPage</class>
 <widget class="QWidget" name="ClientSimuPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="port"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>CPC Host</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cpcHost"/>
     </item>
     <item>
      <widget class="QPushButton" name="cpcConnBtn">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>SMPS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="smpsHost"/>
     </item>
     <item>
      <widget class="QPushButton" name="smpsConnBtn">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TESTDEV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="testDevHost"/>
     </item>
     <item>
      <widget class="QPushButton" name="testConnBtn">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
